doc/source/release-notes/Percona-Server-5.7.10-1.rst

```diff
@@ -0,0 +1,33 @@
+.. rn:: 5.7.10-1
+
+===========================
+ |Percona Server| 5.7.10-1
+===========================
+
+ Percona is glad to announce the first Release Candidate release of |Percona Server| 5.7.10-1 on December 14th, 2015 (Downloads are available `here <http://www.percona.com/downloads/Percona-Server-5.7/Percona-Server-5.7.10-1/>`_ and from the :doc:`Percona Software Repositories </installation>`).
+
+ Based on `MySQL 5.7.10 <http://dev.mysql.com/doc/relnotes/mysql/5.7/en/news-5-7-10.html>`_, including all the bug fixes in it, |Percona Server| 5.7.10-1 is the current Release Candidate release in the |Percona Server| 5.7 series. All of |Percona|'s software is open-source and free, all the details of the release can be found in the `5.7.10-1 milestone at Launchpad <https://launchpad.net/percona-server/+milestone/5.7.10-rc1>`_
+
+ This release contains all the bug fixes from latest |Percona Server| 5.6 release (currently |Percona Server| `5.6.27-76.0 <http://www.percona.com/doc/percona-server/5.6/release-notes/Percona-Server-5.6.27-76.0.html>`_). 
+ 
+New Features
+============
+
+ |Percona Server| 5.7.10-1 is not available on *RHEL* 5 family of Linux distributions and *Debian* 6 (squeeze).
+ 
+ Complete list of changes between |Percona Server| 5.6 and 5.7 can be seen in :ref:`changed_in_57`.
```

Comment:
Needs a "known issues" section, based on information in BP and some more: 1) PAM plugin broken;  2) innodb_show_verbose_locks / innodb_show_locks_help broken; 3) TokuDB has a number of known issues (https://tokutek.atlassian.net/browse/DB-920?jql=labels%20%3D%205.7) ( @georgelorchpercona might have a wording for "known issues" section here)